app/controllers/comments_controller.rb

```diff
@@ -115,6 +115,35 @@ def create
     render json: { error: message }, status: :unprocessable_entity
   end
 
+  def moderator_create
+    return if RateLimitChecker.new(current_user).limit_by_action("comment_creation")
+
+    response_template = ResponseTemplate.find(params[:response_template][:id])
+    authorize response_template, :moderator_create?
+
+    moderator = User.find(SiteConfig.mascot_user_id)
+    @comment = Comment.new(permitted_attributes(Comment))
+    @comment.user_id = moderator.id
+    @comment.body_markdown = response_template.content
+    @comment.moderator_id = current_user.id
```

Comment:
Sorry for the late comment I just noticed this, it is a little confusing. I'm assuming that is to track who created the moderated comment. 
```
@comment.user_id = moderator.id
# then 
@comment.moderator_id = current_user.id
```

What if instead of adding a new column we use Audit logging here like @jacobherrington did in this PR https://github.com/thepracticaldev/dev.to/pull/6797/files to track internal stuff.

Reply:
Ahh yeah great point! I copied and pasted this from my old PR 😅. Will update to use audit logging.